packages/front-end/src/app/stores/orgs.ts

```diff
@@ -1,57 +1,47 @@
 import { Organization } from '@easy-genomics/shared-lib/src/app/types/easy-genomics/organization';
-import { OrganizationUserDetails } from '@easy-genomics/shared-lib/src/app/types/easy-genomics/organization-user-details';
-import { OrganizationAccessDetails } from '@easy-genomics/shared-lib/src/app/types/easy-genomics/user';
 import { defineStore } from 'pinia';
-import useUser from '@FE/composables/useUser';
 
 interface OrgsStoreState {
-  selectedUser: OrganizationUserDetails | undefined;
-  selectedOrg: Organization | undefined;
+  // indexed by orgId
+  orgs: Record<string, Organization>;
 }
 
 const initialState = (): OrgsStoreState => ({
-  selectedUser: undefined,
-  selectedOrg: undefined,
+  orgs: {},
 });
 
 const useOrgsStore = defineStore('orgsStore', {
   state: initialState,
 
   actions: {
-    setSelectedUser(user: OrganizationUserDetails) {
-      this.selectedUser = user;
-    },
-
-    setSelectedOrg(org: Organization) {
-      this.selectedOrg = {
-        ...org,
-      };
-    },
-
     reset() {
       Object.assign(this, initialState());
     },
-  },
 
-  getters: {
-    // return the selected user's display name
-    getSelectedUserDisplayName(): string {
-      return String(
-        useUser().displayName({
-          preferredName: this.selectedUser?.PreferredName || '',
-          firstName: this.selectedUser?.FirstName || '',
-          lastName: this.selectedUser?.LastName || '',
-          email: this.selectedUser?.UserEmail || '',
-        }),
-      );
+    async loadOrgs(): Promise<void> {
+      const { $api } = useNuxtApp();
+      const freshOrgs: Record<string, Organization> = {};
+
+      for (const org of await $api.orgs.list()) {
```

Comment:
`TS18046: $api is of type unknown`

Reply:
where does that error come up? I'm not seeing it in vscode.